packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -109,6 +111,19 @@ function ListView({
   const { formatMessage } = useIntl();
   const hasDraftAndPublish = options?.draftAndPublish || false;
   const fetchClient = useFetchClient();
+  const reviewWorkflowColumns = useEnterprise(
+    REVIEW_WORKFLOW_COLUMNS_CE,
+    async () =>
+      (
+        await import(
+          '../../../../../ee/admin/content-manager/pages/ListView/ReviewWorkflowsColumn/constants'
+        )
+      ).REVIEW_WORKFLOW_COLUMNS_EE,
+    {
+      enabled: !!options?.reviewWorkflows,
+    }
+  );
```

Comment:
Ah, so you would separate injecting the header column from the actual column in the table body? If this is the only place where the `cellFormatter` is used I am ok with that, but I'd like to discuss as a follow-up how we can keep them together, because otherwise I am afraid we are scattering things too much.

Will try out your idea and ping you again. Thanks for the help!

Reply:
I've done that now in 009f770e54f6b9780d170f0bfd14111888be067b and would love to hear your and @joshuaellis opinion. I am conflicted. Very happy to see `cellFormatter` go away, but the split between the header definition and a tbody column doesn't feel good.

Reply:
I hear your concern on the split between the header data definition and the UI that renders it but in my opinion this decoupling brings some advantages:
- It's closer to how we are rendering native types: [`CellContent`](https://github.com/strapi/strapi/blob/0f362e3dcf939722d2617b3166fe367a6576c52a/packages/core/upload/admin/src/components/TableList/CellContent.js#L13)
- It's much easier to read and understand what is being rendered and when
- It's easier to debug since you don't have to chase down a data structure to find it's accompanying component.

Reply:
I agree with Mark, i'd favour composability and it's a lot clearer imo.